tests/pass-dep/shims/libc-getentropy.rs

```diff
@@ -0,0 +1,11 @@
+//@only-target-freebsd
```

Comment:
Why should this test run only on freebsd? The function seems to exist on other Unixes as well.

Reply:
because I was focusing on freebsd only. we might be able to expand it in a later PR.

Reply:
I've done this in https://github.com/rust-lang/miri/pull/3162